models/member.js

```diff
@@ -32,7 +38,17 @@ const memberSchema = mongoose.Schema({
         required: true
     },
 
-    groupChat: {
+    signal: {
+        type: Boolean,
+        required: true
+    },
+
+    telegram: {
+        type: Boolean,
+        required: true
+    },
+
+    whatsapp: {
         type: Boolean,
         required: true
     }
```

Comment:
Could this be a single integer? We can use bit flags to get/set the chat.

Reply:
What about now? Bitwise operations will be made in controller for form submissions so, it wasn't complex to implement here.